libobs/obs-windows.c

```diff
@@ -316,8 +316,23 @@ static void log_security_products_by_type(IWSCProductList *prod_list, int type)
 			continue;
 		}
 
-		blog(LOG_INFO, "\t%S: %s (%s)", name,
-		     get_str_for_state(prod_state), get_str_for_type(type));
+		size_t name_length = SysStringLen(name);
+		char *utf8_name = NULL;
+		size_t utf8_name_length =
+			os_wcs_to_utf8(name, name_length, NULL, 0);
+
+		if (utf8_name_length) {
+			utf8_name = bmalloc(utf8_name_length);
+			os_wcs_to_utf8(name, name_length, utf8_name,
+				       utf8_name_length);
+		}
+
+		if (utf8_name) {
```

Comment:
This null check seems unnecessary since the code above already writes to the pointer without checking.

Reply:
review again

Reply:
The check can be removed entirely as `bmalloc` will never return NULL.

Reply:
> The check can be removed entirely as `bmalloc` will never return NULL.
review again

Reply:
> The check can be removed entirely as `bmalloc` will never return NULL.

review?